src/views/Simulation/components/SceneContainer/pixiUtils.js

```diff
@@ -1,57 +1,65 @@
 // Copyright (c) Cosmo Tech.
 // Licensed under the MIT license.
-import * as PIXI from 'pixi.js';
+import { Application, Graphics, GraphicsContext, Container } from 'pixi.js';
 import { getGraphFromInstance } from './graphUtils';
 
-const createTexture = (appRef, options) => {
-  const graphics = new PIXI.Graphics();
-  graphics.beginFill(options.fillColor);
-  graphics.lineStyle(0.75, options.lineColor);
-  graphics.drawRect(0, 0, 8, 8);
-  graphics.endFill();
-  return appRef.current.renderer.generateTexture(graphics, PIXI.SCALE_MODES.LINEAR, 2);
+const createGraphicsContext = (options) => {
+  const grph = new GraphicsContext()
```

Comment:
Do we really need the intermediate `grph` const in this function?
If we need it, then we should probably rename it and use full words for variable names, for readability purposes :slightly_smiling_face: